README.md

```diff
@@ -8,11 +8,13 @@ Popular examples of implementations for this API includes video game system emul
 more generalized 3D programs.
 These programs are instantiated as dynamic libraries. We refer to these as "libretro cores".
 
-![XMB menu driver](http://i.imgur.com/BMR1xxr.png "XMB menu driver")
+![XMB menu driver](https://github.com/libretro/retroarch-scene/blob/master/Scratchpad/images/Versions/1.8.4/XMB-main-menu.jpg?raw=true "XMB menu driver")
```

Comment:
https://github.com/libretro/retroarch-scene ? We could likely upload the screenshots directly to this repo to reference the images by reference.

For example....

```
![XMB Menu Driver](docs/screenshot1.png)
```

Reply:
hmm, does it work like that?

Reply:
You can see a demo of that over at https://github.com/RobLoach/chailove-floppybird ....

```
![Floppy Bird Gameplay](screenshot.png)
[![Floppy Bird Gameplay](screenshot.png)](https://www.youtube.com/watch?v=RLVwTh6qDFI&hd=1)
```

Reply:
Okay I totally missunderstand, I thought we talking docs repo itself instead docs folder in RetroArch.

So it should be okay now https://github.com/libretro/RetroArch/blob/b44968eea57807f623227827a817a868f48b0942/README.md